src/main/java/mate/academy/bookproject/model/Book.java

```diff
@@ -7,26 +7,37 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
 
 @Entity
-@Getter
-@Setter
+@Data

```

Comment:
Don't use `@Data` for entities

```suggestion
@Getter
@Setter
```